src/index.ts

```diff
@@ -107,151 +108,106 @@ export interface CreateGraphOptionsSpec {
   edgeTable: string;
 }
 
-function fileToText(file: File): Promise<string> {
-  return new Promise((resolve, reject) => {
-    const reader = new FileReader();
-    reader.onload = (e) => {
-      if (e.target === null || typeof e.target.result !== 'string') {
-        throw new Error();
-      }
-      resolve(e.target.result);
-    };
-    reader.onerror = (e) => {
-      reject();
-    };
-
-    reader.readAsText(file);
-  });
-}
-
 class MultinetAPI {
-  public client: Client;
+  public axios: MultinetAxiosInstance;
 
   constructor(baseURL: string) {
-    this.client = new Client(baseURL);
+    this.axios = multinetAxiosInstance(axios.create({
+      baseURL,
+    }));
+
   }
 
   public logout() {
-    this.client.get('/user/logout');
+    this.axios.get('/user/logout');
   }
 
-  public userInfo(): Promise<UserSpec | null> {
-    return this.client.get('/user/info');
+  public async userInfo(): Promise<UserSpec | null> {
+    return (await this.axios.userInfo()).data;
```

Comment:
Just want to point out that for these functions, we don't need to manually specify the return types, as far as TypeScript is concerned. It's able to automatically determine the type, since `.data` is typed for each method of `MultinetAxiosInstance`. If you think they should stay to make it a little easier for people using IDEs without strong TypeScript integration, I suppose that's a valid argument.

Reply:
I realized that while I was writing this code, but I erred on the side of keeping the return types to help out people reading the code.

This may all be moot anyway, since we're going to refactor this into a better structure soon. Let's just roll with the verbose type tagging for now.